syntax = "proto3";

message StartRequest {
  int64 car_id = 1;
  string driver_id = 2;
  repeated string passenger_ids = 3;
}
